Documentation/Helm-Charts/operator-chart.md

```diff
@@ -41,28 +41,30 @@ For example settings, see the next section or [values.yaml](https://github.com/r
-| `tolerations`                       | List of Kubernetes `tolerations` to add to the Deployment.                                                                                                | `[]`                                                           |
-| `unreachableNodeTolerationSeconds`  | Delay to use for the node.kubernetes.io/unreachable pod failure toleration to override the Kubernetes default of 5 minutes                                | `5s`                                                           |
-| `currentNamespaceOnly`              | Whether the operator should watch cluster CRD in its own namespace or not                                                                                 | `false`                                                        |
-| `hostpathRequiresPrivileged`        | Runs Ceph Pods as privileged to be able to write to `hostPath`s in OpenShift with SELinux restrictions.                                                   | `false`                                                        |
-| `discover.priorityClassName`        | The priority class name to add to the discover pods                                                                                                       | `<none>`                                                       |
-| `discover.toleration`               | Toleration for the discover pods                                                                                                                          | `<none>`                                                       |
-| `discover.tolerationKey`            | The specific key of the taint to tolerate                                                                                                                 | `<none>`                                                       |
-| `discover.tolerations`              | Array of tolerations in YAML format which will be added to discover deployment                                                                            | `<none>`                                                       |
-| `discover.nodeAffinity`             | The node labels for affinity of `discover-agent` [^1]                                                                                                     | `<none>`                                                       |
-| `discover.podLabels`                | Labels to add to the discover pods.                                                                                                                       | `<none>`                                                       |
+| Parameter                          | Description                                                                                                                | Default        |
```

Comment:
Just change the required line, it looks like the whole table is getting modified here. Try with `vi` editor if you are editing it in vscode or some other editor (with some plugins they auto align everything based on your configurations)

Reply:
Actually, the formatting doesn't change,
But yaa I will try if only specific line got changed in the changes